### PR TITLE
CDAP 3.4.1 Release Notes

### PR DESCRIPTION
**Pull request by awholegunch**
Passes [Quick Build 3](http://builds.cask.co/browse/CDAP-DQB9-3)

[Release Notes](http://builds.cask.co/artifact/CDAP-DQB9/shared/build-3/Docs-HTML/3.4.1-SNAPSHOT/en/reference-manual/release-notes.html#release-3-4-1)

### CODE REVIEW

**Comment by rohitsinha54** [comment]
Quick build passed merging.

**Comment by prinam** [comment]
LGTM. @rohitsinha54 can we merge?

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 1]
```diff
@@ -23,6 +23,60 @@ Cask Data Application Platform Release Notes
...
+
+Known Issues
+------------
+- `CDAP-5900 <https://issues.cask.co/browse/CDAP-5900>`__ - During the
+  upgrade to CDAP 3.4.1, publishing to Kafka is halted because the CDAP
+  Kafka service is not running. As a consequence, any applications that
+  sync to the CDAP metadata will become out-of-sync as changes to the
+  metadata made by the upgrade tool will not be published.
+  
+- See also the *Known Issues* of `version 3.4.0 <#known-issues-340>`_\ .
```
Comment (sreevatsanraman) [inline cdap-docs/reference-manual/source/release-notes.rst]: We decided not link the known issue. Let's add a complete list.

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 2]
```diff
@@ -23,6 +23,125 @@ Cask Data Application Platform Release Notes
...
+- `CDAP-3750 <https://issues.cask.co/browse/CDAP-3750>`__ -
+  If a table schema contains a field name that is a reserved word in the Hive DDL, ``'enable explore'`` fails.
+
+- `CDAP-5900 <https://issues.cask.co/browse/CDAP-5900>`__ - During the
+  upgrade to CDAP 3.4.1, publishing to Kafka is halted because the CDAP
+  Kafka service is not running. As a consequence, any applications that
+  sync to the CDAP metadata will become out-of-sync as changes to the
+  metadata made by the upgrade tool will not be published.
+  
+
```
Comment (rohitsinha54) [inline cdap-docs/reference-manual/source/release-notes.rst:144]: Extra new line. Is it needed ?

Comment (awholegunch) [inline cdap-docs/reference-manual/source/release-notes.rst:144]: Yes. I like extra space (two lines) before a section header.

**Comment by sreevatsanraman** [comment]
Lgtm   after rohit's comments is addressed.

File: cdap-docs/reference-manual/source/release-notes.rst (hunk 2 above)
Comment (rohitsinha54) [inline cdap-docs/reference-manual/source/release-notes.rst:144]: Ok :)

**Comment by rohitsinha54** [comment]
LGTM

**Pull request merged**